tutorials/time-freq/ssvep.py

```diff
@@ -0,0 +1,359 @@
+    subject='02',
+    session='01', task='ssvep'
+) + '_eeg.vhdr'
+
+###############################################################################
+#  Load raw data
+raw = read_raw_bids(bids_filename, bids_root)
+raw.load_data()
+
+###############################################################################
+# Minimal preprocessing
+# ---------------------
+#
+# Due to a generally high SNR in freqeuncy tagging, typical preprocessing steps
+# are considered optional. this doesnt mean, that a proper cleaning would not
+# increase your signal quality!
+#
+# Raw data comes with FCz recording reference, so we will apply common-average
+# rereferencing.
+
+###############################################################################
+# Set montage.
+montage_style = 'easycap-M1'
+montage = mne.channels.make_standard_montage(
+    montage_style,
+    head_size=0.095)  # head_size parameter default = 0.095
+raw.set_montage(montage)
+
+###############################################################################
+# Set common average reference.
+raw.set_eeg_reference('average', projection=False)
+
+###############################################################################
+# Apply notch filtering
+notch = np.arange(raw.info['line_freq'], raw.info['lowpass'] / 2,
+                  raw.info['line_freq'])
+raw.notch_filter(notch, filter_length='auto', phase='zero')
+
+###############################################################################
+# Apply linear filtering
+hp = .1
+lp = 250.
+raw.filter(hp, lp, fir_design='firwin')
+
+
+###############################################################################
+# Frequency analysis
+# ------------------
+# We use Welch's method for frequency decomposition, since it is really fast.
+# You can compare it with, e.g., multitaper to get an impression of the
+# influence on SNR. All the other methods implemented in MNE can be used as
+# well.
+
+###############################################################################
+# Construct epochs.
+events, _ = mne.events_from_annotations(raw)
+raw.info["events"] = events
+tmin, tmax = -1., 30.  # in s
+baseline = None
+epochs = mne.Epochs(raw, events=events, event_id=event_id['12hz'], tmin=tmin,
+                    tmax=tmax, baseline=baseline)
+
+###############################################################################
+# Calculate power spectral density
+tmin = 0.
+tmax = 30.
+fmin = 1.
+fmax = 90.
+sf = epochs.info['sfreq']
+mode_psd = 'welch'  # implemented: 'welch', 'multitaper'
+
+if mode_psd == 'multitaper':
+    psds, freqs = mne.time_frequency.psd_multitaper(
+        epochs, bandwidth=.5,
+        tmin=tmin, tmax=tmax, normalization='full',
+        fmin=fmin, fmax=fmax)
+elif mode_psd == 'welch':
+    psds, freqs = mne.time_frequency.psd_welch(
+        epochs,
+        n_fft=int(sf * (tmax - tmin)), n_overlap=int(sf * .5), n_per_seg=None,
+        tmin=tmin, tmax=tmax,
+        fmin=fmin, fmax=fmax)
```

Comment:
yeah, this doesnt make sense in the static tutorial.. 
we could either eliminate the `if` switch and stick with welch, or compare SNR from both decomposition methods. i didnt do it with this data, but would expect the multitaper to perform worse

Reply:
I've killed the multitaper part.